Automattic/jetpack
# Post List: Fix the stretched thumbnail images.

When we take the thumbnail image from the post content, it might not be
the correct aspect ratio so, as it is displayed in a square, it appears
stretched and distorted.

#### Changes proposed in this Pull Request:

This fixes that by applying `object-fit` to the image. This CSS isn't
supported in IE, but I believe we're no longer supporting IE, and it's a
visual glitch, that doesn't otherwise break the experience.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Enable the post list enhancements
* Create a post with no set featured image, and a rectangular image in the content
* See that the image is stretched when displayed in the post list
* With this branch reload the post list and see that the image is cropped rather than stretched

| Before | After |
|--------|------|
|<img width="302" alt="image" src="https://user-images.githubusercontent.com/96462/140726694-78787e74-ad24-4c63-b310-ceca8dbdf118.png">|<img width="302" alt="image" src="https://user-images.githubusercontent.com/96462/140726985-c3e5564d-ae7d-4e41-bedf-c216bca8da65.png">|
